example/conf/default.yaml

```diff
@@ -25,6 +25,8 @@ logging:
                 notify: true
 assets:
   host: 'http://localhost:8081'
+structure:
```

Comment:
@jgallen23 The example was broken... it couldn't find the layout as it was configured so I added this to fix it.